<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Branch</string>
  </property>
  <widget class="QLineEdit" name="QLE_branchCompany">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Branch Company</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="QLE_branchDescA">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="placeholderText">
    <string>Branch Desc A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_createBranch">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>740</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Branch</string>
   </property>
  </widget>
  <widget class="QLabel" name="QLL_branchDescA">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Branch Desc A</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="QLE_branchCity">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Branch City</string>
   </property>
  </widget>
  <widget class="QLabel" name="QLL_branchTel1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Branch Tel1</string>
   </property>
  </widget>
  <widget class="QLabel" name="QLL_branchCity">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Branch City</string>
   </property>
  </widget>
  <widget class="QLabel" name="QLL_branchCompany">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Branch Company</string>
   </property>
  </widget>
  <widget class="QLabel" name="QLL_branchId">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Branch ID</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CMB_branchId">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="QLL_branchDescE">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Branch Desc E</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="QLE_branchDescE">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="placeholderText">
    <string>Branch Desc E</string>
   </property>
  </widget>
  <widget class="QLabel" name="QLL_branchAddress">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Branch Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="QLE_branchTel1">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Branch Tel1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="QLE_branchAddress">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Branch Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="QLE_branchTel2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>360</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Branch Tel2</string>
   </property>
  </widget>
  <widget class="QLabel" name="QLL_branchTel2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Branch Tel2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="QLE_branchFax">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>410</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Branch Fax</string>
   </property>
  </widget>
  <widget class="QLabel" name="QLL_branchFax">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>410</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Branch Fax</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="QLE_branchEmail">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>470</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Branch Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="QLL_branchEmail">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>470</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Branch Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="QLE_branchNotes">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>520</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Branch Notes</string>
   </property>
  </widget>
  <widget class="QLabel" name="QLL_branchNotes">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>520</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Branch Notes</string>
   </property>
  </widget>
  <widget class="QLabel" name="QLL_branchChangedOn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>570</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Branch Changes on</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="QLE_branchChangedOn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>570</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Branch Changed On</string>
   </property>
  </widget>
  <widget class="QLabel" name="QLL_branchCurrency">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>620</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Branch Currency</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="QLE_branchCurrency">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>620</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Branch Currency</string>
   </property>
  </widget>
  <widget class="QLabel" name="QLL_branchStatus">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>670</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Branch Status</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CMB_branchStatus">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>670</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
